docs/csharp/quick-starts/hello-world.yml

```diff
@@ -0,0 +1,171 @@
+    ```
+
+    Congratulations! You've run your first C# program. Now, replace the two lines above with these:
+- title: Declare and use variables
+  durationInMinutes: 3
+  content: |
+    Your first program printed the `string` "Hello World!" on
+    a screen. The rest of this lesson explores working with
+    the `string` type, which represents text in C#.
+
+    Your first program is limited to printing one message. You can write more
+    useful programs by using **variables**. A **variable** is a symbol you can
+    use run the same code with different values. Let's try it.
+
+    ```csharp
+    string firstName = "Bill";
+    Console.WriteLine(firstName);
+    ```
+
+    Those lines declare a variable, `name` and assigns it a value, "Bill". Then, it
+    prints out the name.
+
+    You can assign different values to any variable you declare. You can change
+    the name to one of your friends. Add these two lines in the interactive window above:
+
+    ```csharp
+    string firstName = "Bill";
+    Console.WriteLine(firstName);
+    firstName = "Maria";
+    Console.WriteLine(firstName);
+    ```
+
+    Notice that the same line of code prints two different messages, based on
+    the value stored in the `firstName` variable.
+
+    You may have also noticed the "Hello" was missing in the last two messages. Let's fix
+    that now.  Modify the last line to the following:
+
+    ```csharp
+    Console.WriteLine("Hello " + firstName);
+    ```
+
+    Click 'Run' again to see the results.
+- title: Work with strings
+  durationInMinutes: 4
+  content: |
+    Your last edit was our first look at what you can do with strings. Let's explore more.
+
+    The `+`, when applied to strings, creates a new string with the text from the first followed by the text from the second. This is referred to as *string concatenation*. You can combine any number of strings, including string variables.  Try this:
+
+    ```csharp
+    string firstName = "Maria";
+    string aFriend = "Sage";
+    Console.WriteLine("My friends are " + firstName + " and " + aFriend);
+    ```
+
+    As you explore more with strings, you can find the length of a string using `Length`.  Add the following code at the bottom of the interactive window:
+
+    ```csharp
+    Console.WriteLine("The name " + firstName + " has " + firstName.Length + " letters");
+    Console.WriteLine("The name " + aFriend + " has " + aFriend.Length + " letters");
+    ```
+
+    > [!TIP]
+    > This is a good time to explore on your own. You've learned that `Console.WriteLine()`
+    > writes text to the screen. You've learned how to declare variables and
+    > concatenate strings together. Experiment in the interactive window. Write
+    > some more messages yourself. Build strings using the `+` operator.
+- title: Do more with strings
+  durationInMinutes: 4
+  content: |
+    Another useful method is Trim, and related methods TrimStart and TrimEnd. These methods remove leading and trailing spaces:
```

Comment:
you haven't introduced methods yet so it shouldn't be another useful method. I'd also add that methods can perform actions or something like that and then show Trim and co.

Reply:
Another idea might be to link here: https://docs.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/classes-and-objects#methods